README.md

```diff
@@ -22,6 +22,18 @@ The GoCD server will store all configuration, pipeline history database, artifac
 docker run -v /path/to/godata:/godata -v /path/to/home-dir:/home/go gocd/gocd-server
 ```
 
+> **Note:** Ensure that `/path/to/home-dir` and `/path/to/godata` is accessible by the `go` user in container (`go` user - uid 1000).
+
+## Installing plugins
+
+All plugins can be installed under `/godata`.
+
+```
+mkdir -p /path/to/godata/plugins/external
+curl --location --fail https://example.com/plugin.jar > /path/to/godata/plugins/external/plugin.jar
+chown -R 1000 /path/to/godata/plugins
```

Comment:
`config.git` is an internal implementation of the server. People should not be providing a `config.git` to the container — they should just provide `/godata` from an existing container.